tensorflow-sys/build.rs

```diff
@@ -164,43 +166,34 @@ fn install_prebuilt() {
     println!("cargo:rustc-link-lib=dylib={}", FRAMEWORK_LIBRARY);
     println!("cargo:rustc-link-lib=dylib={}", LIBRARY);
     let output = PathBuf::from(&get!("OUT_DIR"));
-    let new_framework_library_full_path = output.join(&framework_library_file);
-    if new_framework_library_full_path.exists() {
-        log!(
-            "File {} already exists, deleting.",
-            new_framework_library_full_path.display()
-        );
-        std::fs::remove_file(&new_framework_library_full_path).unwrap();
-    }
-    let new_library_full_path = output.join(&library_file);
-    if new_library_full_path.exists() {
+
+    let framework_files =
+        glob::glob(&format!("{}/lib*.{}", lib_dir.display(), DLL_EXTENSION)).unwrap();
```

Comment:
This misses libtensorflow.so.1.15.0 and libtensorflow.so.1, and libtensorflow.so.1 is what's being linked against.  Likewise for libtensorflow_framework.

Reply:
Indeed the code above, It will take libtensorflow.1.15.0.so but not libtensorflow.so.1.15.0. I did not expect to have the version after the extension.

Reply:
I have removed glob dependency and list all file in lib_dir.